qdrant-landing/content/blog/datatalk-club-podcast-plug.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Navigating challenges and innovations in search technologies"
+draft: false
+slug: navigating-challenges-innovations
+short_description: Podcast on search and LLM with Datatalk.club
+description: Podcast on search and LLM with Datatalk.club
+preview_image: /blog/navigating-challenges-innovations/preview/preview.png
+date: 12.01.2024T15:39:53.751Z
```

Comment:
Hugo (doc tool) expects a different date format. This makes the article link appear at https://qdrant.tech/blog/

```suggestion
date: 2024-01-12T15:39:53.751Z
```